content/about/globalteam/index.en.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Global Team"
+weight: 1
```

Comment:
@shannonpileggi, with the weight you can control the position in the menu. It currently has 1, just like Mission, so it put this one first because G < M